turtlebot4.tf

```diff
@@ -4,6 +4,7 @@ locals {
     "roni-kreinin",
     "tfoote",
     "tonybaltovski",
+    "civerach-cpr"
```

Comment:
Please put it in in alphabetical order, and add a trailing comma so future merges are easier.

The CI will probably catch this but it's not triggering promptly right now.

Reply:
I applied these edits to the PR to unblock it.